src/cpu/x64/jit_brgemm_conv.cpp

```diff
@@ -1814,8 +1816,8 @@ void brgemm_convolution_fwd_t<isa>::maybe_conv_weights(const exec_ctx_t &ctx,
         const auto out_kh_offs
                 = nb_rd * jcp.oc_block * wei_dsz * jcp.vnni_block;
 
-        parallel_nd(
-                jcp.ngroups, jcp.nb_oc, KH, [=](dim_t g, dim_t ocb, dim_t kh) {
+        parallel_nd(jcp.ngroups, jcp.nb_oc, KH,
+                [= WA_THIS_COPY_CAPTURE](dim_t g, dim_t ocb, dim_t kh) {
```

Comment:
(nit)
It's more of a compatibility feature than a workaround so maybe name it accordingly?